app/controllers/admin/basic_projects_controller.rb

```diff
@@ -20,4 +20,50 @@ def index
       @csv_mode = false
     end
   end
+
+  def show
+    @project = BasicProject.find(params[:id])
+    authorize @project
+    prep_form_vars
+  end
+
+  def destroy
+    @project = BasicProject.find(params[:id])
+    authorize @project
+
+    if @project.destroy
+      redirect_to admin_basic_projects_path, notice: I18n.t(:notice_deleted)
+    else
+      prep_form_vars
+      render :show
+    end
+  end
+
+  def update
+    @project = BasicProject.find(params[:id])
+    authorize @project
+    @project.assign_attributes(basic_project_params)
+
+    if @project.save
+      redirect_to admin_basic_project_path(@project), notice: I18n.t(:notice_updated)
+    else
+      prep_form_vars
+      render :show
+    end
+  end
+
+  private
+
+  def prep_form_vars
+    @agent_choices = person_policy_scope(Person.in_division(selected_division).where(has_system_access: true)).order(:name)
```

Comment:
I believe the more conventional way to do this is `policy_scope(Person).in_division...`. I see this `person_policy_scope` stuff is being used in this app but it's not necesary to create these extra methods as Pundit gives you `policy_scope`: https://github.com/elabs/pundit#scopes

Would be great if you could refactor all these please.